packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -343,6 +396,9 @@ function WorklistDataGrid(props) {
     selectRows(false, rowId, rowIndex);
   };
 
+  // -------------------------------------
+  // event handlers
+
```

Comment:
We should have proper comments for the functions instead of these block comments.

Reply:
These are meant to be section headers for organization since there are a lot of existing functions. Each function can have additional separate comments, but that's not what these headers are meant for. If comments for individual functions are needed, they should be added in a separate PR.